tests/h/views/panels_test.py

```diff
@@ -52,13 +53,45 @@ def test_navbar_includes_search_query(pyramid_request):
     assert result['q'] == 'tag:question'
 
 
+@pytest.mark.usefixtures('routes')
+def test_navbar_includes_search_url_when_on_user_search(pyramid_request):
+    pyramid_request.authenticated_user = None
+
+    type(pyramid_request.matched_route).name = PropertyMock(return_value='activity.user_search')
```

Comment:
This doesn't seem to work. It looks like Pyramid sets the `matched_route` in the router which doesn't get called in our tests.